tests/runner/.eslintrc.json

```diff
@@ -12,7 +12,8 @@
 			"globals": {
 				"fetch": false,
 				"Promise": false,
-				"require": false
+				"require": false,
+				"Set": false
```

Comment:
I didn't think so either, but the ecma version was set and they were still throwing errors.

Reply:
Because setting ecmaVersion does not set the environment - but setting the environment does set the ecmaVersion. You need to add the environment. You can the optionally drop ecmaVersion but I guess it may be easier to migrate to flat config later if it stays.

Reply:
But the environment is also set using `node: true` above.

Reply:
Not this environment; the ECMAScript one. `es2022` or whichever version we can rely on.

Reply:
Ah, ok, I'm not used to using more then `node` or `browser`.

Reply:
The ecmaVersion was already set to `2022`, but that still didn't add globals. However, using `es6: true` in the env does work (there is no env for es2022). I like the flat way of doing this better, but this should work for now.